swift/vapor-framework/config.yaml

```diff
@@ -1,6 +1,6 @@
 framework:
```

Comment:
why rename ? (btw, not opposed :stuck_out_tongue: )

Reply:
It's a bug in SwiftPM where it complains if you have a dependency with the same name as the directory you're in annoyingly! Tracked as [SR-12634](https://bugs.swift.org/browse/SR-12634)

Reply:
good to know :stuck_out_tongue:

Reply:
maybe it's a bad idea to `vapor-framework` to display in results -> because the framework is named **vapor**

I understand there is an issue on `SwiftPM`, but its probably a better idea to have a special flag in `config.yaml` (I should have the right to push on this `PR` to do so)